docs/config/index.md

```diff
@@ -1619,3 +1619,11 @@ By default Vitest will run all of your test cases even if some of them fail. Thi
 - **Version:** Since Vitest 0.32.3
 
 Retry the test specific number of times if it fails.
+
+### diff
+
+- **Type:** `string`
+- **CLI:** `--diff=<value>`
+- **Version:** Since Vitest 0.34.4
+
+Path to a diff config that will be used to generate diff interface. Useful if you want to customize diff display.
```

Comment:
Can you add a code example here? To showcase how to define it using typescript. A little bit of information how colors are defined can also be useful (need start/end property I think).